create/create.js

```diff
@@ -8,24 +8,36 @@ import {
 const form = document.querySelector('.bunny-form');
 const logoutButton = document.getElementById('logout');
 
-form.addEventListener('submit', async e => {
+form.addEventListener('submit', async (e) => {
     // prevent default
-
+    e.preventDefault();
     // get the name and family id from the form
+    const data = new FormData(form);
 
+    const bunny = {
+        name: data.get('bunny-name'),
+        family_id: data.get('family-id')
+    };
     // use createBunny to create a bunny with this name and family id
-    
+
+    await createBunny(bunny);
     form.reset();
+    location.replace('/families');
```

Comment:
love how this redirects when you add a bunny, great thinking through the user experience!